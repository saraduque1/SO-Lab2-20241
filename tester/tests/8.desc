Prueba b√°sica del comando path
