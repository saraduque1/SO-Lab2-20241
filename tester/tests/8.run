./wish -c "path /bin /usr/bin && ls"
