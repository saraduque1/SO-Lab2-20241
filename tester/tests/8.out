README.md
comandos.txt
comandos.txt:Zone.Identifier
run-tests.sh
run-xv6-command.exp
tests
tests-out
wish
wish.c
wish.c:Zone.Identifier
wish:Zone.Identifier
xv6-edit-makefile.sh
